CONTRIBUTING.md

```diff
@@ -21,14 +21,14 @@ Contributors should follow the following process:
 7. Create a pull request for your changes on the PrestaShop project. Be sure to follow [the contribution guidelines][2] in your pull request. If you need help to make a pull request, read the [Github help page about creating pull requests][3].
 8. Wait for one of the core developers either to include your change in the codebase, or to comment on possible improvements you should make to your code.
 
-That's it: you have contributed to this open source project! Congratulations!
+That's it: you have contributed to this open-source project! Congratulations!
 
 You don't exactly know where to start? Pick one of our [good first issues][9] to get the list of all beginner-friendly improvements and bugs to fix. Read more about this label on [Build][10].
 
-The PrestaShop documentation features a thorough explanation of the [complete process to your first pull request][4].
+The PrestaShop documentation features a thorough explanation of the [complete process of your first pull request][4].
```

Comment:
This might be intended, as in "the process taking you through the journey, from reading to your pull request"

wdyt @Julievrz ?

Reply:
I agree with you.